### PR TITLE
AA: add ttrpc client tool

### PR DESCRIPTION
**Pull request by Xynnn007**
This tool is useful when we do debugging and test things for ttrpc-AA.

### CODE REVIEW

File: attestation-agent/attestation-agent/src/bin/ttrpc-aa-client.rs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                .expect("request to AA");
+            let token = String::from_utf8(res.Token).unwrap();
+            println!("{token}");
+        }
+        Operation::ExtendRuntimeMeasurement(extend_runtime_measurement_args) => {
+            let req = ExtendRuntimeMeasurementRequest {
+                Domain: extend_runtime_measurement_args.domain,
+                Operation: extend_runtime_measurement_args.operation,
+                Content: extend_runtime_measurement_args.content,
+                ..Default::default()
```
Comment (mkulke) [inline attestation-agent/attestation-agent/src/bin/ttrpc-aa-client.rs:142]: Can we add an (optional) arg for the register?

**Review by mkulke — COMMENTED** [review]
thanks, this will be very useful for debugging. will this be covered by the CI?

File: attestation-agent/attestation-agent/src/bin/ttrpc-aa-client.rs (hunk 1 above)
Comment (Xynnn007) [reply attestation-agent/attestation-agent/src/bin/ttrpc-aa-client.rs:147]: Added.

**Comment by Xynnn007** [comment]
@mkulke Now cargo clippy and fmt check will cover the tool code.

**Review by mkulke — APPROVED** [review]

**Pull request merged**